### PR TITLE
Rebuild app

### PR DESCRIPTION
**Pull request by mateuszobara02**
- Remove vendors
- Create responsive page design
- Get rid of static data
- Fetch data from GitHub
- Update README.md

### CODE REVIEW

**Comment by piecioshka** [comment]
Wow! What a huge change!

@mateuszobara can you provide link to demo app? I would like to see what app it looks like after your change.

**Comment by mateuszobara02** [comment]
@piecioshka Sure!
https://mateuszobara.github.io/popularity-of-javascript-frameworks/

**Comment by piecioshka** [comment]
@mateuszobara You should know that this project is used simple-data-table intentionally.
This project is a real-life demo of simple-data-table.
I cannot merge changes that remove simple-data-table.

**Review by piecioshka — CHANGES_REQUESTED** [review]
@mateuszobara You should know that this project is used simple-data-table intentionally.
This project is a real-life demo of simple-data-table.
I cannot merge changes that remove simple-data-table.

**Comment by mateuszobara02** [comment]
@piecioshka I've added simple-data-table to the project.

File: src/vendors/simple-data-table/src/index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ class SimpleDataTable {
         this.addButtonLabel = options.addButtonLabel || '✚';
         this.defaultColumnPrefix = options.defaultColumnPrefix || 'column';
         this.defaultColumnNumber = options.defaultColumnNumber || 3;
+        this.defaultHighlightedCellClass = options.defaultHighlightedCellClass || 'highlighted-cell';
```
Comment (piecioshka) [inline src/vendors/simple-data-table/src/index.js:8]: Vendors canont be modified.

**Review by piecioshka — CHANGES_REQUESTED** [review]

File: src/vendors/simple-data-table/src/index.js (hunk 1 above)
Comment (mateuszobara02) [reply src/vendors/simple-data-table/src/index.js:8]: Vendors are not modified. I updated 'simple-data-table' to the latest version.

**Comment by piecioshka** [comment]
@mateuszobara Where we have pie chart?

**Comment by mateuszobara02** [comment]
@piecioshka I decided to remove pie chart because I also added the forks count. One additional chart will make this page irredible. In my opinion this app is more user-friendly and easier to read without these charts. I also think that the table with hex code looks weird.

**Comment by piecioshka** [comment]
Thanks @mateuszobara02 for your help! 🍺

**Pull request merged**